ppr-ui/package.json

```diff
@@ -1,13 +1,13 @@
 {
   "name": "ppr-ui",
-  "version": "3.0.6",
+  "version": "3.0.7",
   "private": true,
   "appName": "Assets UI",
   "sbcName": "SBC Common Components",
   "scripts": {
     "dev": "vite dev",
     "build": "vite build",
-    "serve": "vite preview --port 8080",
+    "preview": "vite preview --port 8080",
```

Comment:
Where would this be used?

Reply:
When you want to preview the production ready build